framework/math/quadratures/angular/triangle_quadrature.h

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: 2024 The OpenSn Authors <https://open-sn.github.io/opensn/>
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "framework/math/quadratures/angular/angular_quadrature.h"
+
+namespace opensn
+{
+
+class TriangleQuadrature : public AngularQuadrature
+{
+protected:
+  unsigned int method_;
+  unsigned int sn_;
+  unsigned int moments_;
+
+  void TriangleInit();
```

Comment:
check with @wdhawkins but we may need fo explicit names such as TriangleQuadratueInitialize

Reply:
Just go with `Initialize` for right now.